src/stylesheets/elements/_typography.scss

```diff
@@ -10,8 +10,20 @@ body {
 }
 
 p {
-  margin-top: 0;
+  @include fontsize(18);
```

Comment:
Remove `fontsize()` here and in desktop. Use `@include zeta` Instead, everything else can stay